src/app/app.module.ts

```diff
@@ -25,6 +26,11 @@ import { AddUserPage } from './admin/add-user/add-user.page';
 import { AddCategoryPage } from './admin/add-category/add-category.page';
 import { AddProductPage } from './admin/add-product/add-product.page';
 import { AddTKProductPage } from './admin/add-tk-product/add-tk-product.page';
+import { ReactiveFormsModule } from '@angular/forms';
+import { DashboardService } from '../../src/providers/services/dashboard/dashboard.service'
```

Comment:
Don't import service here, in service file add @Injectable({providedIn: 'root'})